src/main/scala/za/co/absa/commons/config/ConfigurationImplicits.scala

```diff
@@ -22,86 +22,86 @@ import org.apache.commons.lang.StringUtils.isNotBlank
 import scala.util.Try
 
 /**
-  * The object contains extension methods for the [[org.apache.commons.configuration.Configuration Configuration]] interface.
```

Comment:
Let's make some agreement on indentation.
Currently the majority of the rest javadocs in the project use two space chars indent. Only `BuildInfo` and `NoEmptyValuesSupport` are exceptions.

Reply:
That was just automatic formatter. Ok two spaces then.